autosklearn/pipeline/components/data_preprocessing/one_hot_encoding/minority_coalescer.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import scipy.sparse
+
+import autosklearn.pipeline.implementations.MinorityCoalescer
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import CategoricalHyperparameter, \
+    UniformFloatHyperparameter
+from ConfigSpace.conditions import EqualsCondition
+
+from autosklearn.pipeline.components.base import AutoSklearnPreprocessingAlgorithm
+from autosklearn.pipeline.constants import *
+from autosklearn.util.common import check_for_bool
+
+
+class MinorityCoalescer(AutoSklearnPreprocessingAlgorithm):
+    """ Group together categories which occurence is less than a specified minimum fraction.
+    """
+
+    def __init__(self, use_minimum_fraction=True, minimum_fraction=0.01, random_state=None):
+        self.use_minimum_fraction = use_minimum_fraction
+        self.minimum_fraction = minimum_fraction
+        
+
+    def _fit(self, X, y=None):
+        self.use_minimum_fraction = check_for_bool(self.use_minimum_fraction)
+        if self.use_minimum_fraction is False:
+            self.minimum_fraction = None
+        else:
+            self.minimum_fraction = float(self.minimum_fraction)
+
+        self.preprocessor = autosklearn.pipeline.implementations.MinorityCoalescer\
+            .MinorityCoalescer(minimum_fraction=self.minimum_fraction)
+
+        return self.preprocessor.fit_transform(X)
+
+    def fit(self, X, y=None):
```

Comment:
This function isn't necessary, could you please rename the function above to fit?

Reply:
ok